pareto/strategic_water_management/strategic_produced_water_optimization.py

```diff
@@ -997,12 +1012,9 @@ def create_model(df_sets, df_parameters, default={}):
     )
 
     model.p_kappa_Pipeline = Param(
-        model.s_L,
-        model.s_L,
-        model.s_D,
-        default=5,
-        initialize=df_parameters["PipelineExpansionCost"],
-        doc="Pipeline construction/expansion capital cost for selected increment [$/bbl]",
+        default=120000,
+        initialize=df_parameters["PipelineExpansionCost"]["pipeline_expansion_cost"],
+        doc="Pipeline construction/expansion capital cost for selected increment [$/inchmile]",
```

Comment:
Can we change this unit to "[$/inch-mile]"? That's how I've seen folks express this.